lua/orgmode/objects/help.lua

```diff
@@ -201,7 +205,18 @@ function Help._prepare_orgagenda(mappings, max_height)
   end
 
   local content = { string.format(' **Orgmode mappings - Agenda%s:**', scroll_more_text), '' }
-  for _, item in ipairs(helps.orgagenda) do
+  if vim.b.org_agenda_type == 'agenda' then
```

Comment:
Since `for` block is actually the same as for `generic` mappings, we can just merge both generic and agenda specific into one, and let the "generic" loop handle everything. Something like this:

```lua
-- Copy to make sure we do not override any references
local mappings = vim.deepcopy(helps.orgagenda.generic)
if vim.b.org_agenda_type == 'agenda' then
  mappings = utils.concat(helps.orgagenda.agenda_specific, mappings)
end

for _, item in ipairs(mappings) do
  -- ...
endfor
```

Reply:
Great idea :)

Reply:
Note that I forgot to copy "agenda_specific" mappings, since `utils.concat` will mutate it.

Reply:
Should be fixed. Also made some changes regarding the replacement of `<prefix>` with the actual prefix from the config.